packages/page-pattern-modal/src/components/pattern-selector-item.tsx

```diff
@@ -6,25 +6,39 @@ interface PatternSelectorItemProps {
 	onSelect: ( value: string ) => void;
 	patternPostID: number | null;
 	title?: string;
+	stylesheet: string;
 	theme: string;
 	value?: string;
 }
 
+const PATTERN_PREVIEW_ENDPOINT = 'https://public-api.wordpress.com/wpcom/v2/block-previews/pattern';
+
+const PATTERN_SOURCE_SITE = 'dotcompatterns.wordpress.com';
+
+const getPatternPreviewUrl = (
+	stylesheet: string,
+	language: string,
+	patternId: string
+): string => {
+	const params = new URLSearchParams( {
+		wpcom_site: window._currentSiteId.toString(),
```

Comment:
It should be `demo_site`, not `wpcom_site`.

Reply:
yes, I'll update it